applications/StructuralMechanicsApplication/tests/response_function_tests/FD_strain_energy_response_parameters.json

```diff
@@ -0,0 +1,22 @@
+{
+    "kratos_response_settings":{
+        "response_type"          : "global_finite_differencing",
+        "sensitivity_model_part_name" :"Parts_AREAS",
+        "nodal_sensitivity_variables"  : ["SHAPE"],
+        "element_sensitivity_variables"  : [],
```

Comment:
If the response type is one of the adjoint response functions it would be necessary to define these things also within  the ```sub_response_settings```.

Reply:
Yes, the `sub_response_settings` should contain the "original" settings for the response. In case of an adjoint, it will contain the adjoint response settings.